### DISCUSSION TITLE
Would it be Beneficial To have Example Usage for CharacterVirtual

### DISCUSSION THREAD

**Discussion by cuppajoeman** [jrouwe/JoltPhysics#870] [Ideas]
@jrouwe I was wondering if it'd be appropriate to contribute a section in the docs that talks about how to correctly instantiate a `CharacterVirtual` and control it. Let me know your thoughts

**Comment by jrouwe** [comment]
How about [this](https://jrouwe.github.io/JoltPhysics/#character-controllers)?

**Reply by cuppajoeman** [reply]
Hey I read the new section and yes that helps a lot.

I was able to create a character, but I got a segfault until I added a `mShape` to the instantiated `CharacterVirtualSettings`. From what I understand that occurs because all the options for the settings have a default value except for the shape attribute.

Also from the examples I think we actually give it a velocity through `SetLinearVelocity` to get it moving, but I'm not sure

It might be a good idea to add a note, that you at least have to specify a shape when you create the settings.
